server/src/main/java/org/elasticsearch/index/termvectors/TermVectorsService.java

```diff
@@ -241,7 +242,7 @@ private static Analyzer getAnalyzerAtField(IndexShard indexShard, String field,
         } else {
             return mapperService.indexAnalyzer(
                 field,
-                f -> { throw new IllegalArgumentException("No analyzer configured for field " + f); }
+                f -> Lucene.KEYWORD_ANALYZER    // if no analyzer configured it must be untokenized so return a keyword analyzer
```

Comment:
Are there scenarios where an analyzer is not available, but the field is NOT something like `_ignored` where this error is valid? I don't know enough to help make that decision. It just seems like we through an error for some purpose, and now we are not.

Reply:
I gotcha, I also see (thanks to you) that we already check if the field is a StringType field and that it is indexed. 

Thanks!